filebeat/harvester/log.go

```diff
@@ -265,6 +267,6 @@ func (h *Harvester) close() {
 		h.file.Close()
 		logp.Debug("harvester", "Stopping harvester, closing file: %s", h.Path)
 	} else {
-		logp.Debug("harvester", "Stopping harvester, NOT closing file as file info not available: %s", h.Path)
+		logp.Warn("harvester", "Stopping harvester, NOT closing file as file info not available: %s", h.Path)
```

Comment:
This normally happens, when a file cannot be opened for example because filebeat does not have the access rights. The error what exactly happened should be printed out previously when the harvester tried to open the file.

I'm also worried about us leaking file descriptions are there seem to be especially on windows related to this, but so far I haven't seen this log messages in the logs when it happened.